.expeditor/scripts/shared.ps1

```diff
@@ -72,15 +81,17 @@ function Install-RustToolchain($Toolchain) {
 function Wait-Supervisor($Timeout=1) {
   Write-Host "Waiting up to $Timeout seconds for Supervisor to start..."
   $startTime = [DateTime]::Now
-  $success = (Test-NetConnection -ComputerName localhost -Port 9631).TcpTestSucceeded
+  # Test-NetConnection could be used here, but it is not implemented in PowerShell core
+  # See https://github.com/PowerShell/PowerShell/issues/6932
+  $success = (Test-Connection -ComputerName localhost -TCPPort 9631).TcpTestSucceeded
   while(!$success) {
       Start-Sleep -Seconds 1
       $timeTaken = [DateTime]::Now.Subtract($startTime)
       if($timeTaken.TotalSeconds -ge $Timeout) {
           Write-Error "Timed out waiting $Timeout seconds for Supervisor to start"
           break
       }
-      $success = (Test-NetConnection -ComputerName localhost -Port 9631).TcpTestSucceeded
+      $success = (Test-Connection -ComputerName localhost -TCPPort 9631).TcpTestSucceeded
```

Comment:
See above. We might want to extract the Test-Connection business to a separate function.